### PR TITLE
add c# client FreeRedis

### PR DESCRIPTION
**Pull request by 2881099**

### CODE REVIEW

File: clients.json [hunk 1]
```diff
@@ -1875,5 +1875,15 @@
...
     "authors": [],
     "active": true
+  },
+  
+  {
+    "name": "FreeRedis",
+    "language": "C#",
+    "url": "https://github.com/2881099/FreeRedis",
+    "description": "This .NET client supports redis6.0+, cluster, sentinel, pipeline, And simple api.",
+    "authors": ["2881099"],
```
Comment (itamarhaber) [inline clients.json:1885]: This should contain one or more Twitter handles. If you don't use it, just leave it empty likeso:
```suggestion
    "authors": [],
```

File: clients.json [hunk 2]
```diff
@@ -1875,5 +1875,15 @@
...
+    "recommended": true,
```
Comment (itamarhaber) [inline clients.json:1886]: This is reserved - please remove.
```suggestion
```

**Comment by 2881099** [comment]
Thanks!

My name is YeXiangqin. Before that, I had another redis client open source project for 4 years.

CSRedisCore： https://github.com/2881099/csredis

CSRedisCore is based on csredis project.

----

FreeRedis is a new project. Its goal is to solve all the problems encountered in the past.

Thanks!

**Comment by itamarhaber** [comment]
Dear @2881099 - please apply my review so I can merge this.

**Comment by 2881099** [comment]
Processed, thank you

**Review by itamarhaber — APPROVED** [review]

**Comment by 2881099** [comment]
https://github.com/redis The client contains multiple languages, but no C#.

Can FreeRedis join the Redis organization（ https://github.com/redis ) ?

Thanks.

**Pull request merged**